dvc/remote/base.py

```diff
@@ -142,6 +138,18 @@ def _check_requires(self, config):
         ).format(url, missing, " ".join(missing), self.scheme)
         raise RemoteMissingDepsError(msg)
 
+    def _get_checksum_jobs(self, repo, config):
+        checksum_jobs = config.get(Config.SECTION_REMOTE_CHECKSUM_JOBS)
+        if not checksum_jobs:
+            try:
+                core = repo.config.config.get(Config.SECTION_CORE)
+                checksum_jobs = core.get(
+                    Config.SECTION_CORE_CHECKSUM_JOBS, self.CHECKSUM_JOBS
+                )
+            except (AttributeError, ConfigError):
```

Comment:
Could you please clarify what `AttributeError` is for?

Reply:
there are places in **tests** that pass `None` or custom `Repo()` to `RemoteBASE` so such way I am checking that case.
https://github.com/JIoJIaJIu/dvc/blob/42c34d7269c24187a09bbebe0469647862a75fdd/tests/unit/remote/test_gdrive.py#L27

Reply:
Why not `if self.repo` then? Attribute error might be too generic, as it will catch something that might happen deeper. Your first implementation had both `if self.repo` as well as `except AttributeError`, right? Let's keep only `if self.repo` then.

Reply:
> Why not if self.repo then

Cause here there is a repo but not config, https://github.com/JIoJIaJIu/dvc/blob/42c34d7269c24187a09bbebe0469647862a75fdd/tests/unit/remote/test_gdrive.py#L27
I didn't dare to touch the test 
so I need to check `self.repo` and `self.repo` can be without config. I can change it to
```
self.repo.get('config', {})
```
Should I?

Reply:
That specific test will still have repo.config though, because of https://github.com/iterative/dvc/blob/master/dvc/repo/__init__.py#L82 . Maybe some other test is not mocking it properly, then it should be fixed, unless it is unreasonably hard, of course 🙂

Reply:
@JIoJIaJIu , I thought I had left a comment about `AttributeError`, but I couldn't find it.
Do you remember if I posted it? Am I going cuckoo :see_no_evil: ?

Reply:
@mroutis https://github.com/iterative/dvc/pull/3133#discussion_r366151735

Reply:
For avoiding checking `config` attribute existence I extended `Repo` class in `GDrive` tests
https://github.com/JIoJIaJIu/dvc/commit/aec7781c7cc9f7d0fa4b61ce7d70c60c4f4f15ba#diff-b780ac32bee8290565619b80d7051c85R18
Could you please double check that it doesn't break anything?

Reply:
@mroutis [I've resolved](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#resolving-conversations) it marking that the changes were applied. Is it ok or better to keep all threads open?

Reply:
@JIoJIaJIu, it's fine to resolve them, I do the same) Just couldn't find it this time :sweat_smile: